### ISSUE TITLE
Obstipation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9792]
**Preferred term label:**
Obstipation

**Synonyms**
Lack of flatulence, no gas passage, tympanic distended abdomen

**Definition (free text, please give PubMed ID)**
Lack of flatulence, no gas passage, tympanic distended abdomen

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Intestinal obstruction

**Your nano-attribution (ORCID)**
